include/crypto/riscv_arch.h

```diff
@@ -13,19 +13,28 @@
 # include <ctype.h>
 # include <stdint.h>
 
-# define RISCV_DEFINE_CAP(NAME, INDEX, BIT_INDEX) +1
+# if defined(OPENSSL_SYS_LINUX) && !defined(FIPS_MODULE)
+#  if __has_include(<asm/hwprobe.h>)
+#   define HWPROBE
```

Comment:
The define name seems a little bit too terse to me. Perhaps `OSSL_RISCV_HWPROBE` ?

Reply:
Changed to that name.